pkg/server/cloud/cloud.go

```diff
@@ -109,6 +109,7 @@ type Image struct {
 	Name         string
 	RootDevice   string
 	CreationTime *time.Time
+	VolumeDiskSize *int32 // in GiB
```

Comment:
Random idea, doesn't have to be done.  This could also be an `int32` instead of an `*int32`. It would default to 0 and, since volumes can't have a size of zero, the zero sized voulume would be equivalent to a nil value if you were to use a pointer.

The only real advantage of this is that you don't have to test for a non-nil value before using the value in the comparison in `if *image.VolumeDiskSize > volSizeGiB`.  Also, you don't need to dereference the value.

Reply:
I originally used pointer, as I thought it's pretty standard way to deal with optional values in struct. However, defaulting to 0 makes total sense to me. Thanks!